gmail/src/views/create_task.ts

```diff
@@ -70,7 +70,7 @@ export function buildCreateTaskView(state: State, query: string = "", hideCreate
         noProject = !state.searchedProjects.length;
     }
 
-    const odooServerUrl = State.odooServerUrl;
+    const odooServerUrl = PropertiesService.getUserProperties().getProperty("ODOO_SERVER_URL");
```

Comment:
why not using the getter ? (I would prefer to keep the getter so it centralize the property name)

Reply:
Yeah I thought about adding a util or a service but since PropertiesService is already a service I thought it might be a bit much to add a file just for that, since there doesn't really seem to be any file where this would belong.

I'll add one if that's fine with you though

Reply:
there's already one
```
static get odooServerUrl(): string {
```

Reply:
arf ok :/ you can keep it like that then

Reply:
can't we remove the getter so ?

Reply:
Yeah, I guess so. I also moved the setter for consistency.

I'm not moving everything related to userproperties though, since the commit is not really about that